kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/thirdChallenge/FlightFinder.java

```diff
@@ -28,7 +35,7 @@ public void scanFlightsFromAirport(String startAirport) {
         }
     }
 
-    public void scanFlightsToAirport(String endAirport) {
+    public void findTo(String endAirport) {
 
         if (SearchChecker.isFlightEndInThisAirport(endAirport).size() > 0) {
```

Comment:
szczerze mowiac, to nie wiem, czy te warunki sa w ogole konieczne. zastosuj filtrowanie w przebiegu strumieniowym.